data-raw/render_italy_codes.R

```diff
@@ -0,0 +1,13 @@
+# italy codes
+italy_codes <- vroom::vroom("data-raw/italy_codes.csv")
+italy_codes <- tibble::tibble(
```

Comment:
all looks sensible but I don't think the field info is needed here (as it should go in the class)